### ISSUE TITLE
Suggestion for API

### ISSUE THREAD

**Issue by jumson** [ucig/bible-api#11]
**related to a problem?**
You do not show code for API implementation/server.

**Describe the solution you'd like**
How are you serving this up? Or at the least, check out this API from API.bible and consider whether you can improve or expand that one, or make your own swagger reference.
https://api.scripture.api.bible/v1/swagger.json

**Comment by McSnurtle** [comment]
I agree that proper dedicated docs - even a github wiki page - would be reaaaally nice (with response examples, etc)
